src/components/questions/questions.module.css

```diff
@@ -76,16 +71,62 @@ li {
   list-style-type: none;
 }
 
-/* May use these later.... commented out for now until cleanup */
 
-/*
 
-.answerList {
+/* Forms CSS
+----------------------------------------------------
+*/
 
+input[type=text] {
```

Comment:
There are so many css selectors. I had to look up what the [] did. Seems like it could be pretty helpful for me going forward!